cartridges/int_algolia/cartridge/scripts/algolia/customization/productModelCustomizer.js

```diff
@@ -93,29 +93,45 @@ function createAlgoliaLocalizedCategoryObject(category) {
     return result;
 }
 
+var NEW_ARRIVALS_CATEGORY_ID = 'newarrivals';
+var specialValueHandlers = {
+    newArrivalsCategory: function(productModel) {
+        if (!empty(productModel.categories)) {
+            for (var i = 0; i < productModel.categories.length; i += 1) {
+                var rootCategoryId = productModel.categories[i][productModel.categories[i].length - 1].id;
```

Comment:
Maybe we give it a name as `lastCategoryId` as it refers to the ID of the last category in the chain